vsts/scripts/copyAllBlobsToProdStorageAccount.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+
+set -ex
+
+declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && cd .. && pwd )
+source $REPO_DIR/platforms/__common.sh
+
+azCopyDir="/tmp/azcopy-tool"
+
+function blobContainerExistsInProd() {
+	local containerName="$1"
+	local exitCode=1
+	curl -I "$PROD_SDK_STORAGE_BASE_URL/$containerName?restype=container" 2> /tmp/curlError.txt 1> /tmp/curlOut.txt
+	grep "HTTP/1.1 200 OK" /tmp/curlOut.txt &> /dev/null
+	exitCode=$?
+	rm -f /tmp/curlOut.txt
+	rm -f /tmp/curlError.txt
+	if [ $exitCode -eq 0 ]; then
+		return 0
+	else
+        return 1
+	fi
+}
+
+function copyBlobContainerToProd() {
+    local platformName="$1"
+
+    if shouldOverwriteSdk || shouldOverwritePlatformSdk $platformName; then
+        echo
+        echo "Overwriting blob container '$platformName' in storage account '$destinationSdk'."
+        # azcopy copy [source] [destination] [flags]
+        if [ $dryRun == "false" ] ; then
+            "$azCopyDir/azcopy" copy \
+                "$sourceSdk/$platformName$sasToken" \ 
+                "$PROD_SDK_STORAGE_BASE_URL/$platformName$PROD_STORAGE_SAS_TOKEN" --overwrite true --recursive
```

Comment:
Any reason we need the `--recursive` flag? I don't see that being used for the dev --> prod script we have today, so just curious what it's needed for here